From 7a4f9ca8cd512603ad0772626918a9c2716571f8 Mon Sep 17 00:00:00 2001
From: Karthik Ramanan <a0393906@ti.com>
Date: Thu, 15 Jan 2015 16:54:38 +0530
Subject: [PATCH] gstreamer1.0: Include the files into the targetfs

Signed-off-by: Karthik Ramanan <a0393906@ti.com>
---
 .../gstreamer/gst-plugins-package.inc              |    1 +
 1 file changed, 1 insertion(+)

diff --git a/meta/recipes-multimedia/gstreamer/gst-plugins-package.inc b/meta/recipes-multimedia/gstreamer/gst-plugins-package.inc
index 0503b34..0fb1191 100644
--- a/meta/recipes-multimedia/gstreamer/gst-plugins-package.inc
+++ b/meta/recipes-multimedia/gstreamer/gst-plugins-package.inc
@@ -51,6 +51,7 @@ PACKAGES += "${PN}-apps ${PN}-meta ${PN}-glib"
 FILES_${PN}-apps = "${bindir}"
 
 FILES_${PN} = "${datadir}/gstreamer-${LIBV}"
+FILES_${PN} += "${libdir}/gstreamer-1.0/*.so"
 FILES_${PN}-dbg += "${libdir}/gstreamer-${LIBV}/.debug"
 FILES_${PN}-glib = "${datadir}/glib-2.0"
 
-- 
1.7.9.5

